testing/ncids-css-testing/stories/design-tokens/typography/fonts/fonts.scenarios.js

```diff
@@ -65,4 +65,36 @@ module.exports = [
 			},
 		],
 	},
+	{
+		label: 'NCI Headings',
+		storyId: 'design-tokens-typography-fonts-role-heading--nci-heading',
+		viewports: [
+			{
+				label: 'desktop',
+				width: 1024,
+				height: 768,
+			},
+			{
+				label: 'tablet',
```

Comment:
from the scss media queries we need a test for default (mobile) and tablet-lg. I'm seeing desktop and tablet on this file being generated too. is this supposed to be mobile and are both desktop breakpoints supposed to be tablet-lg?